controls/V-219279.rb

```diff
@@ -64,5 +64,15 @@
   tag fix_id: 'F-21003r305166_fix'
   tag cci: ['SV-109885', 'V-100781', 'CCI-000172']
   tag nist: ['AU-12 c']
-end
 
+  if os.arch == 'x86_64'
+    describe auditd.syscall('finit_module').where { arch == 'b64' } do
+      its('action.uniq') { should eq ['always'] }
+      its('list.uniq') { should eq ['exit'] }
+    end
+  end
```

Comment:
move this block to the bottom to make the pattern clear that the 64 bit arch checks both 32 and 64 bit rules